### PR TITLE
wzhzhzh

### PR DESCRIPTION
**Pull request by atteria**
## wzhzhzh
wzhzhzh

## wzhzhzh
wzhzhzh

## wzhzhzh
wzhzhzh

image

## wzhzhzh
:cl: wzhzhzh
tweak: wzhzhzh
/:cl:

### CODE REVIEW

**Review by AffectedArc07 — APPROVED** [review]
wzhzhzh

File: code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm [hunk 1]
```diff
@@ -244,6 +244,7 @@
 	if(stat & (NOPOWER|BROKEN))
 		return
 	start()
+	atom_say("wzhzhzh")
```
Comment (hal9000PR) [inline code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm:247]: wzhzhzh?

Comment (atteria) [reply code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm:247]: wzhzhzh...

Comment (Contrabang) [reply code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm:247]: wzhzhzh!

Comment (Pyrodes) [reply code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm:247]: wzhzhzh?!

**Comment by procdrone** [comment]
hzhzhzw

**Pull request merged**